test/dlc_tests/ec2/tensorflow/inference/test_tensorflow_inference.py

```diff
@@ -303,13 +303,6 @@ def host_setup_for_tensorflow_inference(
     is_graviton=False,
     model_name=None,
 ):
-    # Wait for any existing apt-get calls to finish before moving on
-    # TODO(Mike Schneider): Improve this by adding a check for running apt-get processes and wait for them to finish,
-    # then timeout after a given amount of time if other apt-get calls are taking too long.
-    ec2_connection.run((f"sleep 180"), hide=True)
-
-    # Install PIP so we can test
-    ec2_connection.run((f"sudo apt-get update && sudo apt-get install -y python3-pip"), hide=True)
```

Comment:
Removed this because we install python and pip through pyenv as soon as the instance is created.

https://github.com/aws/deep-learning-containers/blob/a82ff809a94bf893541f7634c985d10e51cc0628/test/dlc_tests/conftest.py#L772

Running `apt-get` commands seems to cause dpkg lock issues (which should already have been resolved by killing the unattended upgrades service during pyenv install), and skipping this redundant python3-pip installation can avoid flakiness from the dpkg lock unavailability.